### DISCUSSION TITLE
Accessing Commit History & StudentPack

### DISCUSSION THREAD

**Discussion by gatimugabriel** [community/community#128528] [GitHub Education]
Hello there. I lost access to my account recovery codes and github mobile authenticator (for this, I had logged out before). So in the recovery section, all actions led me to unlink my email account and create a new account with it, and I did so. However, on the new account, the commit history has not shown up. Moreover, I needed to access github education, using the student pack, but on trying to register, I got an error prompt that the school email that I used to access the pro benefits was already added to the old account.
Note: I have already unlinked the two email addresses, the old email and the school email from that account.

**Comment by sreemanthink** [comment]
Responce #10113
